release.py

```diff
@@ -0,0 +1,224 @@
+"""Script that prepares a new release of a version."""
+
+import argparse
+import json
+import logging
+import re
+import tempfile
+from functools import cached_property
+from itertools import product
+from pathlib import Path
+
+import git
+import tomli
```

Comment:
is this another toml-lib? we have another one imported in django_evaluation/settings/local.py. Why different ones?

Reply:
The tomli lib is buildin from python 3.10 onwards. I also find it unfortunate that they didn't name it toml.

Reply:
Ok, I didn't know that. Then we should switch to it in local.py as well at some point